docs/user-guide/llamaedge-lobechat.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 5
+---
+
+# LobeChat + LlamaEdge
+
+The LobeChat framework supports OpenAI providers, and since LlamaEdge is compatible with the OpenAI API, you can use LlamaEdge as the backend Large Language Model (LLM) API. This integration supports:
+
+* The hosted LobeChat service
+* Any product built on the open-source LobeChat framework
+
+## Prerequisites
+
+Before proceeding, follow the [Getting started with LlamaEdge guide](get-started-with-llamaedge.md) to run an open-source LLM locally.
+
+## Steps to integrate Dify and LlamaEdge
+
+Open the [LobeChat Language Model setting page](https://chat-preview.lobehub.com/settings/modal?agent=&session=inbox&tab=llm&topic=CIfo1UYZ) and choose the first one OpenAI.
+
+First, Enter some random characters in the OpenAI API Key field, and input `http://localhost:8080/v1` in the API Proxy Address field.
+
+Then, enable the Client-Side Fetching Mode. 
+
+Next, Click on the Get Model List button to automatically detect the model you're using. Select that model from the list.
+
+Finnaly, you can click on the Check button to check the connect status.
+
+![](lobechat-llamaedge-01.png)
```

Comment:
Please upload the image

Reply:
I see that you uploaded the image to main. In the future, please upload the image in this PR. Thanks.